docs/presto-admin-commands.rst

```diff
@@ -331,6 +331,25 @@ once, you could run ::
     sudo ./presto-admin connector add
     sudo ./presto-admin server restart
 
+Adding a Custom Connector
+--------------------------
+In order to install a custom connector not included with Presto, the jar must be
+added to the Presto plugin location using the ``plugin add_jar`` command before
```

Comment:
re: 'before': the order of plugin add_jar and connector add doesn't matter, but this is fine.  They have to be executed in some order